tests/system/test_system.py

```diff
@@ -52,7 +53,7 @@ def _get_credentials_and_project():
     return credentials, project
 
 
-@pytest.fixture(scope=u"module")
+@pytest.fixture(scope="module")
```

Comment:
Blacken must have done this.

Reply:
'SOK, especially given that firestore is now PY3-only.

Reply:
Wait, was that from running `nox -e blacken`?  Because that doesn't remove the `u` prefixes on my machine.

Reply:
Yes. Here's what I happens when I reintroduce one and then run `nox -e blacken`:

~~~~
(firestore) chris@spirit:~/proj/google/firestore$ git diff
diff --git a/tests/system/test_system.py b/tests/system/test_system.py
index e68e31c..68a2a7b 100644
--- a/tests/system/test_system.py
+++ b/tests/system/test_system.py
@@ -53,7 +53,7 @@ def _get_credentials_and_project():
     return credentials, project
 
 
-@pytest.fixture(scope="module")
+@pytest.fixture(scope=u"module")
 def client():
     credentials, project = _get_credentials_and_project()
     yield firestore.Client(project=project, credentials=credentials)
(firestore) chris@spirit:~/proj/google/firestore$ nox -e blacken
nox > Running session blacken
nox > Creating virtual environment (virtualenv) using python3.6 in .nox/blacken
nox > pip install black==19.10b0
nox > black docs google tests noxfile.py setup.py
reformatted /home/chris/proj/google/firestore/tests/system/test_system.py
All done! ✨ 🍰 ✨
1 file reformatted, 97 files left unchanged.
nox > Session blacken was successful.
(firestore) chris@spirit:~/proj/google/firestore$ git diff
(firestore) chris@spirit:~/proj/google/firestore$ 
~~~~